src/containers/BooksForm.js

```diff
@@ -0,0 +1,66 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { createCreator } from '../actions/index';
+
+function mapDispatchToProps(dispatch) {
+  return {
+    addBook: bookObj => dispatch(createCreator(bookObj)),
+  };
+}
+
+class BooksForm extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      title: '',
+      category: 'Category',
+    };
+    this.handleChanges = this.handleChanges.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleChanges(event) {
+    if (event.target.id === 'title') {
+      this.setState({
+        title: event.target.value,
+      });
+    } else if (event.target.id === 'category') {
+      this.setState({
+        category: event.target.value,
+      });
+    }
+  }
+
+  handleSubmit(event) {
+    event.preventDefault();
+    const { addBook } = this.props;
+    addBook(this.state);
+    this.setState({
+      title: '',
+      category: 'Category',
+    });
```

Comment:
You could add a form reset after this line to reset the form after submitting the data.